js/controller.js

```diff
@@ -1137,6 +1137,11 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
       this.state.configLoaded = true;
       this.renderSkin();
       this.createPluginElements();
+
+      if (params.showAdControlsOverAds === true) {
```

Comment:
Looks ok but wouldn't it be better to use the same mechanism that is used for VAST in order to show the controls?